stdlib/2.7/socket.pyi

```diff
@@ -283,36 +283,17 @@ class socket:
     # --- methods ---
     # second tuple item is an address
     def accept(self) -> Tuple['socket', Any]: ...
-
-    @overload
-    def bind(self, address: tuple) -> None: ...
-    @overload
-    def bind(self, address: str) -> None: ...
-
+    def bind(self, address: Union[tuple, str]) -> None: ...
     def close(self) -> None: ...
-
-    @overload
-    def connect(self, address: tuple) -> None: ...
-    @overload
-    def connect(self, address: str) -> None: ...
-
-    @overload
-    def connect_ex(self, address: tuple) -> int: ...
-    @overload
-    def connect_ex(self, address: str) -> int: ...
-
+    def connect(self, address: Union[tuple, str]) -> None: ...
+    def connect_ex(self, address: Union[tuple, str]) -> int: ...
     def detach(self) -> int: ...
     def fileno(self) -> int: ...
 
     # return value is an address
     def getpeername(self) -> Any: ...
     def getsockname(self) -> Any: ...
-
-    @overload
-    def getsockopt(self, level: int, optname: str) -> str: ...
-    @overload
     def getsockopt(self, level: int, optname: str, buflen: int) -> str: ...
```

Comment:
Actually this needs to keep the overload. But the original was also wrong: return type for `getsockopt(int, str)` is int, not str.